src/style.css

```diff
@@ -1,3 +1,89 @@
-body {
+/* #region ZERO */
+body{
+  padding: 0;
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. You can add a generic font family as a fallback, such as `font-family: Roboto, Arial, sans-serif;`.